src/Apps/Artwork/Components/ArtistInfo.tsx

```diff
@@ -1,62 +1,156 @@
-  MarketInsightsProps,
-  SelectedExhibitions,
-  SelectedExhibitionsProps,
+  ArtistBioFragmentContainer as ArtistBio,
+  MarketInsightsFragmentContainer as MarketInsights,
+  SelectedExhibitionFragmentContainer as SelectedExhibitions,
 } from "Styleguide/Components"
-import { EntityHeader } from "Styleguide/Components/EntityHeader"
 
-interface ArtistInfoProps
-  extends MarketInsightsProps,
-    SelectedExhibitionsProps {
-  name: string
-  bio: string
-  credit?: string
+interface ArtistInfoProps {
+  artist: ArtistInfo_artist
+  user: User
+  mediator?: Mediator
 }
 
-export class ArtistInfo extends React.Component<ArtistInfoProps> {
-  render() {
-    // FIXME: Fetch from relay
-    const props = {
-      href: "/artist/francesca-dimattio",
-      imageUrl: "https://picsum.photos/110/110/?random",
-      name: "Francesca DiMattio",
-      meta: "American, b. 1979",
+export const ArtistInfo: SFC<ArtistInfoProps> = props => {
+  const imageUrl = get(props, p => p.artist.image.cropped.url)
+
+  return (
+    <>
+      <StackableBorderBox p={2} flexDirection="column">
+        <EntityHeader
+          name={props.artist.name}
+          meta={props.artist.formatted_nationality_and_birthday}
+          imageUrl={imageUrl}
+          FollowButton={
+            <FollowArtistButton
+              artist={props.artist}
+              user={props.user}
+              onOpenAuthModal={() => {
+                props.mediator.trigger("open:auth", {
+                  mode: "signup",
+                  copy: `Sign up to follow ${props.artist.name}`,
+                  signupIntent: "follow artist",
+                  afterSignUpAction: {
+                    kind: "artist",
+                    action: "follow",
+                    objectId: props.artist.id,
+                  },
+                })
+              }}
+              render={({ is_followed }) => {
+                return (
+                  <Sans
+                    size="2"
+                    weight="medium"
+                    style={{
+                      cursor: "pointer",
+                      textDecoration: "underline",
+                    }}
+                  >
+                    {is_followed ? "Following" : "Follow"}
+                  </Sans>
+                )
+              }}
+            >
+              Follow
+            </FollowArtistButton>
+          }
+        />
+        <Spacer mb={1} />
+        <ArtistBio bio={props.artist} maxChars={300} />
+      </StackableBorderBox>
+      <StackableBorderBox p={2}>
+        <MarketInsights artist={props.artist} border={false} />
+      </StackableBorderBox>
+      <StackableBorderBox p={2}>
+        <SelectedExhibitions
+          artistID={props.artist.id}
+          border={false}
+          totalExhibitions={props.artist.counts.partner_shows}
+          exhibitions={props.artist.exhibition_highlights}
+          ViewAllLink={
+            <a href={`https://artsy.net/artist/${props.artist.id}/cv`}>
```

Comment:
should be `www.`

Reply:
I'm gonna set up a shared ENV var in reaction so we don't have to hardcode this stuff anymore...